src/visitor.ts

```diff
@@ -582,6 +582,10 @@ export class GeneratorVisitor extends Visitor {
             return
         }
 
+        if (schema.items) {
+          this.visitSchema(schema.items, null, combiner)
```

Comment:
Any particular reason for the explicit null and not passing in the parent schema param from the function's arguments?